### DISCUSSION TITLE
Official Ring app UI not responding when alarm triggered

### DISCUSSION THREAD

**Discussion by monkeypashion** [tsightler/ring-mqtt#344] [Support Q&A] [answered]
If Home Assistant is used to arm and disarm my Ring alarm, the official app UI updates in sync (as does Homekit and Home Assistant Lovelace UI).  This is great as I can invoke those services in different ways (depending on the scenario) with the exactly the same outcome. However when I run the alarm trigger service the official app UI does not respond.  And in addition the siren does not automatically sound like it does if the alarm is triggered by a Ring sensor.

Ultimately I'd like to use my Hue motion sensors to trigger the alarm, sound the siren and then manage (disarm etc) through the Ring app. Currently I can only do this with the Ring sensors.

Would there be a workaround if this?

**Comment by tsightler** [comment, marked as answer]
I'm not aware of any way to cause the Ring alarm to enter triggered state via an API call (there's no method to trigger the alarm from the app other than panic switches), thus that function is not supported via Home Assistant.  Ring alarm does not support triggering via 3rd party sensors.

If you have a Ring Protect plan you can enable the panic switches which will trigger either police/fire alarm, same as panic toggles available in the app, however, triggering the alarm via non-certified devices is likely a violation of the monitoring terms of service.  Because of this, panic switches have to be explicitly enabled via the configuration options.

**Reply by monkeypashion** [reply]
Thanks for the info - and thanks for all the great work on this project - super value!
